lib/client/workloadendpoint.go

```diff
@@ -194,8 +194,16 @@ func (w *workloadEndpoints) convertKVPairToAPI(d *model.KVPair) (unversioned.Res
 	ah.Spec.InterfaceName = bh.Name
 	ah.Spec.MAC = bh.Mac
 	ah.Spec.Profiles = bh.ProfileIDs
-	ah.Spec.IPNetworks = nets
-	ah.Spec.IPNATs = nats
+	if len(nets) == 0 {
```

Comment:
because it returns empty slice instead of `nil` which is inconsistent from other fields (which return `nil` when they're not initialized). This is the one I discussed with Casey  yesterday